MVCDemos/ViewModels/EditorsViewModel.cs

```diff
@@ -58,6 +63,8 @@ public class EditorsViewModel {
         [VerifyAge(21, ErrorMessage = "You must be at least {1} years old")]
         public DateTime? Date { get; set; }
 
+        [VerifyDateRangeSelection(ErrorMessage = "Both start and end dates must be selected")]
+        [VerifyDateRange(25, ErrorMessage = "The vacation period must not exceed {1} days")]
         public DateTime?[] VacationDates { get; set; }
 
         [DevExtremeRequired(ErrorMessage = "You must agree to the Terms and Conditions")]
```

Comment:
Let's add `Updated` badge in MenuMeta.json.
And don't we need to add new files with view models to `additionalFiles` in MenuMeta?